dev/PowerNotifications/PowerNotificationsPal.h

```diff
@@ -14,7 +14,7 @@ struct CompositeBatteryStatus
 
 DECLARE_HANDLE(CompositeBatteryStatusRegistration);
 HRESULT GetCompositeBatteryStatus(CompositeBatteryStatus* compositeBatteryStatusOut);
-typedef void (*OnCompositeBatteryStatusChanged)(CompositeBatteryStatus compositeBatteryStatus);
+typedef void (*OnCompositeBatteryStatusChanged)(CompositeBatteryStatus &compositeBatteryStatus);
```

Comment:
Mixed feelings about this one. This is describing the public surface of the closed-source ABI, and we've historically made the OS's public Win32 APIs C-compatible. Not sure if we intend to follow the same principle for the Reunion closed-source ABI; if so, this would need to be a pointer and not a reference. @jonwis, thoughts?

Considering it non-blocking for this PR. If we do need to make this header/ABI C-compatible, we can clean up in a subsequent PR.